cmd/crio/main.go

```diff
@@ -514,6 +540,10 @@ func main() {
 			Usage: "specify the GID mappings to use for the user namespace",
 			Value: "",
 		},
+		cli.StringSliceFlag{
+			Name:  "additional-devices",
```

Comment:
see comment about default_devices above - also variables names all around should be defaultDevices or something like that

Reply:
See my disagreement above.